django/db/backends/postgresql/schema.py

```diff
@@ -52,11 +61,7 @@ class DatabaseSchemaEditor(BaseDatabaseSchemaEditor):
     def quote_value(self, value):
         if isinstance(value, str):
             value = value.replace("%", "%%")
-        adapted = psycopg2.extensions.adapt(value)
-        if hasattr(adapted, "encoding"):
-            adapted.encoding = "utf8"
-        # getquoted() returns a quoted bytestring of the adapted value.
-        return adapted.getquoted().decode()
+        return sql.quote(value, self.connection.connection)
```

Comment:
[`psycopg`'s `sql.quote()`](https://www.psycopg.org/psycopg3/docs/api/sql.html#psycopg.sql.quote) accepts `sql` and `context` so we shouldn't pass a connection
```suggestion
        return sql.quote(value)
```